packages/rum-core/src/domain/configuration.ts

```diff
@@ -142,3 +143,25 @@ export function validateAndBuildRumConfiguration(
     baseConfiguration
   )
 }
+
+export function serializeRumConfiguration(configuration: RumInitConfiguration) {
+  const baseSerializedConfiguration = serializeConfiguration(configuration)
+
+  return assign(
+    {
+      premium_sample_rate: configuration.premiumSampleRate,
+      replay_sample_rate: configuration.replaySampleRate,
+      session_replay_sample_rate: configuration.sessionReplaySampleRate,
+      action_name_attribute: configuration.actionNameAttribute,
+      use_allowed_tracing_origins:
+        configuration.allowedTracingOrigins !== undefined ? !!configuration.allowedTracingOrigins : undefined,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IIUC, there are 3 cases: 

1. `allowedTracingOrigins: undefined`
2. `allowedTracingOrigins: []`
3. `allowedTracingOrigins: ['some origin', ..]`

The current implementation will send

1.  `use_allowed_tracing_origins: undefined`
2. `use_allowed_tracing_origins: true`
3. `use_allowed_tracing_origins: true`

IMO, it could be more interesting to send:

1.  `use_allowed_tracing_origins: false` (we could send undefined but do we care about this difference?)
2. `use_allowed_tracing_origins: false`
3. `use_allowed_tracing_origins: true`

wdyt?

(probably same reasoning for `excludedActivityUrls`)

Reply:
Fair point. Seems reasonable